src/main/java/cz/xtf/radanalytics/util/configuration/RadanalyticsConfiguration.java

```diff
@@ -0,0 +1,51 @@
+package cz.xtf.radanalytics.util.configuration;
+
+import cz.xtf.XTFConfiguration;
+
+public class RadanalyticsConfiguration extends XTFConfiguration {
+	private static final String IMAGE_PY_SPARK = "rad.driver.python";
+	private static final String IMAGE_JAVA_SPARK = "rad.driver.java";
+	private static final String IMAGE_SCALA_SPARK = "rad.driver.scala";
+	private static final String IMAGE_OSHINKO_REST = "rad.oshinko.rest";
+	private static final String IMAGE_OSHINKO_CLI = "rad.oshinko.cli";
+	private static final String IMAGE_OSHINKO_WEBUI = "rad.oshinko.webui";
```

Comment:
this property doesn't have a method, if this property is not needed and used, it can be deleted